gtfs-realtime/spec/en/reference.md

```diff
@@ -66,6 +66,7 @@ Fields labeled as **experimental** are subject to change and not yet formally ad
             *   [Effect](#enum-effect)
             *   [TranslatedString](#message-translatedstring)
                 *   [Translation](#message-translation)
+			*	[SeverityLevel](#enum-severitylevel)
```

Comment:
Could you please match the whitespace indenting here to the same level as `TranslatedString`?  It renders right on GitHub but the Markdown should still be formatted consistently.

Reply:
Can you please clarify what you mean by this? If you view the full markdown file, the indentation is formatted correctly.